main.js

```diff
@@ -1,3 +1,44 @@
 import './style.css';
 
-console.log('Hello world');
+document.querySelector(".button2").addEventListener("click", function () {
+    helptip(this);
+});
+
+function helptip(t) {
+    helptip.v = 0;
+    let b = t.children[(t.children.length - 1)];
```

Comment:
It is not good way too work with children as arrays. You can set classNames and use special selectors to work with exact element, that you need